bundle/metadata/dependencies.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- type: olm.package
+  value:
+    packageName: ramen-hub-operator
+    version: 4.11.0
```

Comment:
because of `RAMEN_VERSION ?= 0.0.1`, it should be
```suggestion
    version: 0.0.1
```
here. Please run `make bundle` to update this value.